src/UnitTests/Analysis/OutParameterTransformerTests.cs

```diff
@@ -69,6 +69,11 @@ private void PerformTest(FileUnitTester fut)
 
 				proc.Write(false, fut.TextWriter);
 				fut.TextWriter.WriteLine("====================");
+                //$REVIEW: OutpMutual test failed. OutParameterTransformer
+                // removes phi assignments from uses. Is it correct, John?*/
+                if (proc.Name == "fn0C00_0004")
```

Comment:
I just looked at the `analysis-development` branch, and it turns out the `OutParameterTransformer` class is dead. There is a  `CallRewriterTests.CrwMutual`, currently disabled. I will make sure that passes before my next commit.